### ISSUE TITLE
content: dataQuality

### ISSUE THREAD

**Issue by nathcast** [Rothamsted-Ecoinformatics/eRA#15]
- [x] The content to the dataQuality page was taken from the green site. 

- [x] It still refers to laurence. 

Please update the page 
It is at: web site update > files . WebPages>default>dataQuality.php

- [x] Add a section on Data Quality for met data. Met data instruments are routinely calibrated. DIscuss with Tony Scott. 

- [ ] New Rothamsted Reseach Data Quality statement required, updated from existing one.

**Comment by mglendining** [comment]
Hi Nathalie, I don't think I have access to this .php file.
Margaret

________________________________

**Comment by Perrymas** [comment]
I don't seem to be able to edit the File in Teams either, all though I can open it and comment on it.

**Comment by mglendining** [comment]
Laurence Benjamin's name has now been removed.

**Comment by mglendining** [comment]
The rest of the file is OK, though we don't do double entry any more, but this is how most of the data was done in the past.

**Comment by richardostler** [comment]
There are several superscript numbers in the text but they don't refer to any references on the page. I also think the text on Data checking procedures should be revised to state double entry is no longer performed, but I think we do have programmatic validations on the data?

Finally, the Rothamsted Research Quality statement looks out of date (it has a reference to something which expired in 2018). Can you see if there is an updated version on the Rothamsted website and if not remove the link.

**Comment by mglendining** [comment]
Hi All
Further thought regarding data quality.
The various equipment at the met stations is all calibrated regularly by Campbell's scientific, so we ought to include a statement about that.
Agreed, double entry no longer happens,but very little data is now entered manually. However, when it was, we used double entry.
I'll contact Tony and try to produce a sentence or two, if you think this is the appropriate place?
Margaret
________________________________

**Comment by mglendining** [comment]
I have now added a paragraph about instument calibration at the three met stations. @nathcast @richardostler @Perrymas

**Comment by mglendining** [comment]
The link to docport no longer works, so it is not possible to check all the SOPs listed on this page. Tony is trying to update the one about measuring met data and uploading to e-RA, but now that Laurence Benjaimin has left, this is no longer maintained. @richardostler

**Comment by richardostler** [comment]
@mglendining I'll email Tim James to see if he knows what's going on with it.

**Comment by mglendining** [comment]
@richardostler apparently the old link does work, but as I don't have VPN I can't see it.

**Comment by mglendining** [comment]
@richardostler  @nathcast Elizabeth King (in the safety office) has shown me how to access docport. 
if you go to the new intranet and the 9 little dots arranged in a square on the left hand side of the Rothamsted Research ribbon, it  appears in the 'app launcher' as 'docport'. you don't need VPN for this.

**Comment by mglendining** [comment]
@richardostler  @nathcast Elizabeth King (in the safety office) has shown me how to access docport. 
if you go to the new intranet and the 9 little dots arranged in a square on the left hand side of the Rothamsted Research ribbon, it  appears in the 'app launcher' as 'docport'. you don't need VPN for this.

**Comment by mglendining** [comment]
I have contacted Elizabeth King (Safety Office) about a new Rothamsted Research Quality statement. Otherwise, all done.

**Comment by mglendining** [comment]
Elizabeth King doesn't know, suggested Tim James. emailed Tim James.

**Comment by mglendining** [comment]
@richardostler I think there are Rothamsted Research Quality control statements, but as I am not on VPN, I can't access them. I can nearly see them, then they disappear! eg https://rothamsted.sharepoint.com/sites/QualityAssurance2/SitePages/Quality-Assurance-Guide.aspx Click on the ROthamsted link, and it disappears!

**Comment by mglendining** [comment]
@nathcast  Reply from Tim James: "I am not aware of another more up to date version [than the 2014 statement]. There are a number of issues that need to be resolved within the institute and quality management is something that will be flagged up. I cannot give a timescale for producing a new quality control statement but I will be raising such things at the FRG tomorrow."

**Comment by richardostler** [comment]
Edited this as I had a bit of confusion since the old text hadn't been updated from the newer HTML version. I've now updated and made two minor changes - added a reference to the JCoPR and removed the data quality statement as, as has been pointed out above, it is out of date. Hopefully there will be revised documentation we can refer to coming out of the digital transformation project! 

Closing this issue now
